annonymousauthority/P2PCoder
# Taylor - Signup Form validation & Binding Signup Function

## What type of PR is this? (check all applicable)

- [x] Other: Form Validation & Binding Signup Function
  
  
## Description of what this Pull Request does
- Adding form validation for the signup form
- Binding the Firebase signup function to the form component
- 



## Related Tickets & Documents
-Closes: #113 

  
## QA Instructions, Screenshots, Recording

- ROADBLOCKS
- I changed the Link to a button because I wasn't able to catch the form validation otherwise, it would send me straight to the main page. 
- I lost one aspect of the toggle icon function, when the password is visible in the input and a user erases it completely, the icon no longer goes back to the default state of the open icon. I displayed this in the last part of the gif.

![SignupForm](https://github.com/annonymousauthority/P2PCoder/assets/72883059/1fb6aebf-2b98-4ba3-b5c8-921119b7ee40)
